app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
```

Comment:
It is a good practice to leave spaces between built-in libraries and app-specific imports and list import in alphabetical order